lib/kitchen/driver/openstack.rb

```diff
@@ -82,23 +81,22 @@ def create(state)
             config[:server_name] = default_name
           end
         end
-        config[:disable_ssl_validation] && disable_ssl_validation
+        disable_ssl_validation if config[:disable_ssl_validation]
         server = create_server
         state[:server_id] = server.id
-        info "OpenStack instance <#{state[:server_id]}> created."
-        server.wait_for do
-          print '.'
-          ready?
-        end
-        info "\n(server ready)"
+        info ("OpenStack instance <#{state[:server_id]}> created.")
```

Comment:
Good point, I've removed the first line (created) and just going to use the second.